src/viam/utils.py

```diff
@@ -259,7 +259,7 @@ async def to_thread(func: Callable[_P, _R], *args: _P.args, **kwargs: _P.kwargs)
 
 def from_dm_from_extra(extra: Optional[Dict[str, Any]]) -> bool:
     """Used in modular filter components to get the 'fromDataManagement' value from an extra map."""
-    if extra is None:
+    if extra is None or "fromDataManagement" not in extra:
         return False
 
     return bool(extra["fromDataManagement"])
```

Comment:
```suggestion
    return bool(extra.get("fromDataManagement", False))
```
feels a little more natural to me, but either is fine